Source/WebDriver/LogInitialization.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2024 Igalia S.L.  All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+ * OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "config.h"
+#include "LogInitialization.h"
+
+#include "Logging.h"
+#include <wtf/NeverDestroyed.h>
+#include <wtf/text/WTFString.h>
+
+namespace WebDriver {
+
+#if !LOG_DISABLED || !RELEASE_LOG_DISABLED
+
+class LogChannels final : public WTF::LogChannels {
+public:
+    LogChannels()
+    {
+        m_logChannels = {
+            WEBDRIVER_LOG_CHANNELS(LOG_CHANNEL_ADDRESS)
+        };
+    }
+
+private:
+    String logLevelString() final
+    {
+        return WebDriver::logLevelString();
+    }
+};
+
+WTF::LogChannels& logChannels()
+{
+    static NeverDestroyed<LogChannels> logChannels;
+    return logChannels.get();
+}
+
+WTFLogChannel* getLogChannel(const String& name)
+{
+    return logChannels().getLogChannel(name);
+}
+
+#else
+
+WTFLogChannel* getLogChannel(const String& name)
+{
+    UNUSED_PARAM(name);
```

Comment:
You could omit the parameter name instead of using `UNUSED_PARAM()`:

```suggestion
WTFLogChannel* getLogChannel(const String&)
{
```